static/css/restyle/restyle.less

```diff
@@ -201,13 +202,17 @@ button.link:active span {
     0 416px,
     0 0,
     0 0;
-  // This breaks the background panel wrapper out of the containing box,
-  // allowing it to be full-width. This means the background gradient can
-  // span the entire width of the viewport, even though the parent has a
-  // max-width.
-  margin: 0 0 12px ~"calc(-1 * ((100vw - 100%) / 2))";
-  min-width: 960px;
+  margin: 0 0 12px;
+  min-width: 1018px;
   width: 100vw;
+
+  @media (min-width: 1018px) {
+    // This breaks the background panel wrapper out of the containing box,
+    // allowing it to be full-width. This means the background gradient can
+    // span the entire width of the viewport, even though the parent has a
+    // max-width.
+    margin: 0 0 12px ~"calc(-1 * ((100vw - 100%) / 2))";
```

Comment:
Just thinking through this calculation (I see it hasn't changed, though) shouldn't this have a `+ 12px` in it or something since it's `12px` on the right?